src/AdminConsole/appsettings.Development.json

```
@@ -11,7 +11,7 @@
     "CustomerAuthPlanName": "Professional Test"    
   },
   "ConnectionStrings": {
-    "sqlite": "Data Source=adminconsole_dev.db"
+    "sqlite:admin": "Data Source=adminconsole_dev.db"
```

In JSON I vote to represent it with object syntax.
```suggestion
    "sqlite": {
      "admin": "Data Source=adminconsole_dev.db"
    }
```